pkg/inspector/image-inspector.go

```diff
@@ -499,27 +526,28 @@ func (i *defaultImageInspector) getAuthConfigs() (*docker.AuthConfigurations, er
 	return imagePullAuths, nil
 }
 
-func (i *defaultImageInspector) scanImage(s iiapi.Scanner) ([]byte, []byte, error) {
+// TODO: Move this into openscap package
```

Comment:
@enoodle i started to rewrite this then I realized that switching to interface will break tests ;-) but I think it is worth to do this right once and then we can re-use the interface.